.github/workflows/ci.yaml

```diff
@@ -32,11 +32,17 @@ jobs:
             - uses: actions/setup-python@v2
               with:
                   python-version: '3.10'
-            - name: Install pre-commit
-              run: pip install pre-commit==3.6.0
+            - name: Install CI tools
+              run: |
+                  pip install pre-commit==3.6.0
+
+                  # Install dart from flutter
+                  git clone --depth=1 https://github.com/flutter/flutter.git
```

Comment:
Maybe clone a specific tag so upgrades are a bit more controlled?

Reply:
yes, just moved these lines but might as well do it now. thx